### PR TITLE
#120: API-core-view-subview from md to rst

### PR DESCRIPTION
**Pull request by marcinwrobel1986**
* API/core/view/subview transition from .md to .rst

before (.md) | after (.rst)
:----------------: | :-------------:
image | image

### CODE REVIEW

**Comment by fnrizzi** [comment]
@marcinwrobel1986 the formatting/syntax of the cpp code should NOT be just a code block but should use sphinx

**Comment by marcinwrobel1986** [comment]
@fnrizzi I removed Synopsis as suggested as it was duplicated in Description and used Sphinx domains in Description. Please let me know if anything else should be changed. Thanks!

**Review by fnrizzi — APPROVED** [review]

**Comment by fnrizzi** [comment]
can you post here the old page too?

**Review by fnrizzi — APPROVED** [review]

**Review by ajpowelsnl — APPROVED** [review]
This is fine, in terms of the conversion activity, but the expression could be more straightforward (in future work).

File: docs/source/API/core/view/subview.rst [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+    auto s = subview(view,std::pair<int,int>(5,191),Kokkos::ALL,1);
+
+Creates a ``Kokkos::View`` viewing a subset of another ``Kokkos::View``.
+
+Description
+-----------
+
+.. cppkokkos:function:: template<class ViewType, class ... Args> IMPL_DETAIL subview(const ViewType& v, Args ... args)
+
+Returns a new ``Kokkos::View`` ``s`` viewing a subset of ``v`` specified by ``args...``. The return type of subview is an implementation detail and is determined by the types in ``Args...``.
```
Comment (masterleinad) [inline docs/source/API/core/view/subview.rst:23]:
```suggestion
Returns a new ``Kokkos::View`` ``s`` representing a subset of ``v`` specified by ``args...``. The return type of subview is an implementation detail and is determined by the types in ``Args...``.
```

File: docs/source/API/core/view/subview.rst [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+Header File: ``<Kokkos_Core.hpp>``
+
+Usage
+-----
+
+.. code-block:: cpp
+
+    auto s = subview(view,std::pair<int,int>(5,191),Kokkos::ALL,1);
+
+Creates a ``Kokkos::View`` viewing a subset of another ``Kokkos::View``.
```
Comment (masterleinad) [inline docs/source/API/core/view/subview.rst:16]:
```suggestion
Creates a ``Kokkos::View`` representing a subset of another ``Kokkos::View``.
```

Comment (fnrizzi) [reply docs/source/API/core/view/subview.rst]: fixed!

**Review by masterleinad — APPROVED** [review]

**Pull request merged**